aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -115,5 +115,28 @@ Tensor& set_quantizer_(Tensor& self, ConstQuantizerPtr quantizer) {
   return self;
 }
 
+Tensor quantized_clone(const Tensor& self) {
+  TORCH_CHECK(
+      isQIntType(self.scalar_type()),
+      "Input tensor must have a quantized ScalarType");
+  auto quantizer = get_qtensorimpl(self)->quantizer();
+
+  TORCH_CHECK(quantizer->qscheme() == kPerTensorAffine);
+  auto scale = static_cast<PerTensorAffineQuantizer*>(quantizer.get())->scale();
+  auto zero_point =
+      static_cast<PerTensorAffineQuantizer*>(quantizer.get())->zero_point();
+
+  Tensor dst = at::_empty_affine_quantized(
+      self.sizes(),
+      self.options().dtype(toQIntType(self.scalar_type())),
```

Comment:
why do we do `toQIntType` here?

Reply:
I was following the same structure as the other call to _empty_affine_quantized() on line 88.

Reply:
oh that is creating quantized tensor from a int8/int32 tensor. here the scalr_type is already qint8/qint32 so you don't need to call `toQIntType`